recipes/fwdpp/meta.yaml

```diff
@@ -0,0 +1,35 @@
+about:
+    home: 'https://www.github.com/molpopgen/fwdpp'
+    license: "https://raw.githubusercontent.com/molpopgen/fwdpp/master/LICENSE"
+    summary: "A C++ template library for forward-time population genetic simulation. (doi:  10.1534/genetics.114.165019)"
+
+requirements:
+  build:
+    #Must build with GCC even on OS X
+    #b/c libsequence was built that way.
+    #O/W we end up with "missing symbol"
+    #errors.
+    #These requirements are solely 
+    #to get the examples/unit tests
+    #compiled.
+    - gcc  
+    - zlib
+    - zlib
+    - gsl {{CONDA_GSL}}*
+    - libsequence
+    - boost >=1.60.0
+#  run: 
```

Comment:
Correct--see comments at the top.  This is a C++11 template lib.  The build dependencies are simply for testing.